### PR TITLE
Gv/tranformation v1

### PR DESCRIPTION
**Pull request by gane5hvarma**
Libraries feature
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

---
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-server/543)

### CODE REVIEW

File: config/backend-config/backend-config.go [hunk 1]
```diff
@@ -133,9 +133,10 @@ type SourceRegulationT struct {
 }
 
 type SourcesT struct {
-	EnableMetrics bool      `json:"enableMetrics"`
-	WorkspaceID   string    `json:"workspaceId"`
-	Sources       []SourceT `json:"sources"`
+	EnableMetrics bool       `json:"enableMetrics"`
+	WorkspaceID   string     `json:"workspaceId"`
+	Sources       []SourceT  `json:"sources"`
+	Libraries     LibrariesT `json:"libraries"`
```
Comment (psrikanth88) [inline config/backend-config/backend-config.go:139]: rename these to LibraryIDs as we are getting only the library ID in the source right?

File: go.mod [hunk 2]
```diff
@@ -67,6 +67,7 @@ require (
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0
 	golang.org/x/tools v0.0.0-20200226224502-204d844ad48d // indirect
 	google.golang.org/api v0.19.0
+	google.golang.org/appengine v1.6.5
```
Comment (psrikanth88) [inline go.mod:70]: is this new library we are using? vendor files missing?

File: gateway/gateway.go [hunk 3]
```diff
@@ -866,6 +871,7 @@ func (gateway *HandleT) StartWebHandler() {
 	srvMux.HandleFunc("/v1/alias", gateway.stat(gateway.webAliasHandler))
 	srvMux.HandleFunc("/v1/group", gateway.stat(gateway.webGroupHandler))
 	srvMux.HandleFunc("/health", gateway.healthHandler)
+	srvMux.HandleFunc("/test", testHandler)
```
Comment (psrikanth88) [inline gateway/gateway.go:874]: are we using this in tests or something? Ignore comment if so

**Review by psrikanth88 — APPROVED** [review]

**Review by sivashanmukh — APPROVED** [review]
LGTM

**Pull request merged**